mlflow/tracking/client.py

```diff
@@ -220,12 +479,89 @@ def log_metric(self, run_id, key, value, timestamp=None, step=None):
         :param timestamp: Time when this metric was calculated. Defaults to the current system time.
         :param step: Integer training step (iteration) at which was the metric calculated.
                      Defaults to 0.
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            def print_run_info(r):
+                print("run_id: {}".format(r.info.run_id))
+                print("metrics: {}".format(r.data.metrics))
+                print("status: {}".format(r.info.status))
+
+            # Create a run without any tags. Since these are low-level CRUD operations,
+            # this method will create a run. To end the run, you'll have to
+            # explicitly end it.
+            client = MlflowClient()
+            run = client.create_run("0")
```

Comment:
```suggestion
            run = client.create_run(experiment_id="0")
```

to make it easier to understand what `"0"` represents here.

Reply:
or we could use a variable:

```python
experiment_id = "0"
run = client.create_run(experiment_id)
```

Reply:
Either will work. With latter we can add a comment: Experiment "0" is the Default name of the experiment